CMakeLists.txt

```
@@ -66,8 +66,9 @@ set_default_install_dirs()
 
 # Options and Variants ########################################################
 #
-option(HiPACE_MPI            "Multi-node support (message-passing)"       ON)
-option(HiPACE_OPENPMD        "openPMD I/O (HDF5, ADIOS)"                  ON)
+option(HiPACE_MPI            "Multi-node support (message-passing)"             ON)
+option(HiPACE_OPENPMD        "openPMD I/O (HDF5, ADIOS)"                        ON)
+option(HiPACE_AB5_PUSH       "Use fifth order Adams-Bashforth Plasma Pusher"    ON)
```

Could a more general name be considered? Here's a suggestion, the syntax is based on HiPACE_PRECISION
```suggestion

set(HiPACE_PUSHER_VALUES LEAPFROG AB5)
set(HiPACE_PUSHER AB5 CACHE STRING "Plasma pusher (AB5/LEAPFROG)")
set_property(CACHE HiPACE_PUSHER PROPERTY STRINGS ${HiPACE_PUSHER_VALUES})
if(NOT HiPACE_PUSHER IN_LIST HiPACE_PUSHER_VALUES)
    message(FATAL_ERROR "HiPACE_PUSHER (${HiPACE_PUSHER}) must be one of ${HiPACE_PUSHER_VALUES}")
endif()
```

---

A added this option now